.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -234,6 +251,11 @@
                 Condition="'%(ImportedMauiItem.ItemGroupName)' == 'MauiSplashScreen'" />
         </ItemGroup>
 
+        <!-- Map @(MauiAppIcon) to @(MauiImage IsAppIcon=true) -->
+        <ItemGroup>
+            <MauiImage Include="@(MauiAppIcon)" IsAppIcon="True" />
```

Comment:
`@(MauiAppIcon)` is just an alias. Makes the csproj simpler and easier to change via some UI later.